backend/src/main/scala/cromwell/backend/standard/StandardAsyncExecutionActor.scala

```diff
@@ -1057,6 +1058,12 @@ trait StandardAsyncExecutionActor
     */
   def retryEvaluateOutputs(exception: Exception): Boolean = false
 
+  /**
+   * Checks if the job has run into any cloud quota exhaustion and records it to GroupMetrics table
+   * @param runStatus The run status
+   */
+  def checkAndRecordQuotaExhaustion(runStatus: StandardAsyncRunState): Unit = ()
```

Comment:
Backend agnostic method and each backend can implement how to detect quota exhaustion. Currently only implemented in `PipelinesApiAsyncBackendJobExecutionActor.scala`